src/screens/HistoryListScreen.js

```diff
@@ -1,16 +1,23 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { SafeAreaView, FlatList, StyleSheet } from "react-native";
 
 import { getPrescriptionList } from "../redux/features/prescriptionSlice";
+
 import PrescriptionHistoryCard from "../components/PrescriptionHistoryCard";
 
-const PrescriptionHistoryScreen = () => {
+const PrescriptionHistoryScreen = ({ navigation }) => {
   const isLoading = useSelector(state => state.prescription.isLoading);
```

Comment:
```suggestion
  const isLoading = useSelector((state) => state.prescription.isLoading);
```
useSelector 콜백함수 인자를 괄호로 감싸주세요